packages/prisma/migrations/20241114210330_add_event_type_translation_model/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateEnum
+CREATE TYPE "EventTypeAutoTranslatedField" AS ENUM ('DESCRIPTION');
+
+-- AlterTable
+ALTER TABLE "EventType" ADD COLUMN     "autoTranslateDescriptionEnabled" BOOLEAN NOT NULL DEFAULT false;
+
+-- CreateTable
+CREATE TABLE "EventTypeTranslation" (
+    "id" TEXT NOT NULL,
```

Comment:
@hbjORbj Unfortunately I didn't catch this before. We use "uid" to denote string IDs. This is not consistent with other "id" columns because they are integers.

Reply:
Can you please handle in #17704 17657